webapp/channels/src/sass/responsive/_mobile.scss

```diff
@@ -817,6 +817,12 @@
             .post__link {
                 margin: 0 0 8px;
             }
+
+            &:hover {
```

Comment:
@hmhealey is there a reason we want to add this here? It seems to remove the hover effect on posts in the mobile size. Not sure we need to do that.